flask_ask/core.py

```diff
@@ -605,12 +623,15 @@ def _flask_view_func(self, *args, **kwargs):
 
     def _map_intent_to_view_func(self, intent):
         """Provides appropiate parameters to the intent functions."""
-        view_func = self._intent_view_funcs[intent.name]
-        argspec = inspect.getargspec(view_func)
-        arg_names = argspec.args
-        arg_values = self._map_params_to_view_args(intent.name, arg_names)
-
-        return partial(view_func, *arg_values)
+        try:
+            view_func = self._intent_view_funcs[intent.name]
+            argspec = inspect.getargspec(view_func)
+            arg_names = argspec.args
+            arg_values = self._map_params_to_view_args(intent.name, arg_names)
+            return partial(view_func, *arg_values)
+        except:
+            print("Error: Please add the ", intent.name, " to play to run correctly the skill.")
```

Comment:
So this method should never really be called directly by a user or a developer creating a skill, but rather it is invoked by `_flask_view_func`. Additionally, `_map_intent_to_view_func` will only be called from `_flask_view_func` if the incoming request is of "IntentRequest" type.

You can picture a general outline of flask-ask as `User Speaks` -> `Alexa sends request to flask-ask` -> `Process request within _flask_view_func` -> `invoke user defined functions`.  So the user isn't actually responsible for providing the intent name, rather `_map_intent_to_view_func` is called IF there is an intent name.

Therefore, the intent parameter will always be present during this method call and this method will only be called if there is an intent in the request.

That being said, it doesn't necessarily hurt to provide a way to catch an exception. However when using except clauses you always want to specify the type of `Exception` you are checking for, otherwise your remedy may be trying to fix a different problem than what actually occurred. A blanket except statement also isn't able to provide the user/developer with useful information.

Also, placement of the try/except clauses should target a single source of an error. With this code block you are trying to look up a view function from a dictionary, inspect the arguments of the function, and then calling a whole other function using both the intent and argument names. There are quite a few exceptions that could occur there, but we wouldn't know what caused the error, especially if no specific `Exception` is used in the statement.

Perhaps a more useful error-handling would look something like:
```
try:
    view_func = self._intent_view_funcs[intent.name]

# "intent" object has no name attribrute, this is unlikely to occur
except AttribruteError as e:  
    _dbgdump('The IntentRequest from the Alexa Service was malformed and does not include a name for the intent.')
    return None


 # No view func is stored in self._intent_view_funcs for the given intent name
except KeyError as e: 
    _dbgdump('No intent view function found for Intent {}.' .format(intent.name)
    return None # will result in a 404 Not Found response

argspec = inspect.getargspec(view_func)
arg_names = argspec.args
arg_values = self._map_params_to_view_args(intent.name, arg_names)
return partial(view_func, *arg_values)
```